app/lib/templates.dart

```diff
@@ -528,3 +538,8 @@ class PackageLinks extends PageLinks {
   @override
   String formatHref(int page) => '${_basePath ?? '/packages'}?page=$page';
 }
+
+abstract class LogoUrls {
+  static const String smallDartFavicon = '/static/favicon.ico';
+  static const String flutterLogo32x32 = '/static/img/flutter-logo-32x32.png';
```

Comment:
Agreed. I believe `favicon.ico` is the only one that will be `.ico`, the rest will be `.png` going forward. I'm not even sure that we need to stick with `.ico` anymore, need to do some research before ditching it entirely.